requirements.txt

```diff
@@ -5,5 +5,7 @@ django_compressor==2.1
 djmail==0.11.0
 futures==3.0.3
 premailer==2.9.6
-psycopg2==2.6.1
+psycopg2==2.7.3
 requests==2.9.1
+gunicorn
```

Comment:
- How is this? A `pip freeze` always gives the package version

Reply:
You are to use `waitress`: https://andela.slack.com/archives/C85EWN51C/p1512985077000006

Reply:
Have changed to waitress, Done!